### DISCUSSION TITLE
[Accessibility] Auto focus to the text field when opening edit dialogs

### DISCUSSION THREAD

**Discussion by yezhiyi9670** [stakira/OpenUtau#1348] [Ideas]
There are some dialogs that contain text fields, such as the tempo/speed dialog and the lyrics editing dialog. Currently none of them automatically move the keyboard focus to the text field when opened, so the use has to move the mouse to the text field and click it. This is especially painful when a lot of speed labels (曲速标签) have to be added, for example in a ritardando.

![image](https://github.com/user-attachments/assets/e63e65be-e8b3-4f33-9db4-e3d42820938a)
▲ This dialog does not automatically catch keyboard focus when opened.

![image](https://github.com/user-attachments/assets/e33e4ffe-3681-4a0d-92e2-3583e4326d78)
▲ Really a lot of speed labels.

Simply having the dialog automatically grab the focus can make the editing process much smoother.

**Comment by arx-ein** [comment]
Working on this feature in https://github.com/stakira/OpenUtau/pull/1638
